VG Music Studio/Core/GBA/MP2K/Player.cs

```diff
@@ -256,6 +261,8 @@ void EmulateNote(byte key, byte velocity, byte addedDuration)
                                 {
                                     if (!EventExists(offset))
                                     {
+                                        Debug.WriteLine(trackIndex.ToString());
```

Comment:
You can remove all these debug lines, I accidentally included them in my commit

Reply:
done